src/components/elements/Text.jsx

```diff
@@ -1,6 +1,18 @@
+// Available size options(String):
+//   - 'small'
+//   - 'normal'
+//   - 'large'
+// Available color options(String):
+//   - 'cyan'
+//   - 'black'
+// Available sort options(String):
+//   - 'none'
+//   - 'longUrl'
```

Comment:
This one is weird. I don't think you need it. Neither color. Just add external className prop and use it. In case you need any additional style you will pass it from the parent component.